samaritan/templates/samaritan/views/group_members_view.html

```diff
@@ -20,6 +20,11 @@ <h3>Group: <i id="group-name"></i></h3>
       <button type="button" class="btn btn-info btn-md" id="email-group-members">
         <i class="fa fa-envelope-o fa-fw"></i> Email
       </button>
+      
+      <!-- Button sms modal-->
+      <button type="button" class="btn btn-info btn-md" id="sms-group-members">
```

Comment:
Can you make this button green? use btn-success I believe.

Reply:
Done